src/credential/issuance/06-obtain-credential.ts

```diff
@@ -33,7 +33,11 @@ export type ObtainCredential = (
     appFetch?: GlobalFetch["fetch"];
   },
   operationType?: "reissuing"
-) => Promise<{ credential: string; format: string }>;
+) => Promise<{
+  credential: string;
+  format: string;
+  credential_configuration_id: string;
```

Comment:
Is there a reason we return the `credential_configuration_id`? It's already one of the parameters of `obtainCredential`, along with the `credential_identifier`.

Reply:
Yes, I think it would be cleaner to just have a single iteration of `obtainCredential` + `verifyAndParseCredential` to avoid returning the input parameters.